spec/unit/sandbox/path_list_spec.rb

```diff
@@ -14,8 +14,12 @@ module Pod
         end
         expected = %w(
           Banana.modulemap
+          BananaFramework.framework/Headers/BananaFramework.h
+          BananaFramework.framework/Headers/SubDir/SubBananaFramework.h
           BananaFramework.framework/Versions/A/Headers/BananaFramework.h
           BananaFramework.framework/Versions/A/Headers/SubDir/SubBananaFramework.h
+          BananaFramework.framework/Versions/Current/Headers/BananaFramework.h
+          BananaFramework.framework/Versions/Current/Headers/SubDir/SubBananaFramework.h
```

Comment:
I don't think we want this - these are effectively the same file, `Current` is just a directory symlink to `A`. Would this cause build issues due to "duplicate" headers?

Reply:
any updates to this question?